lib/rules.json

```diff
@@ -908,6 +909,118 @@
                             ],
                             "patPolReq": true,
                             "knownDisclosureNumber": true,
+                            "crIntro": "W3C Candidate Recommendation Snapshot <span class=\"rfc2119\">must</span> include 2 paragraphs in \"Status Of This Document\": <pre xml:space=\"preserve\" style=\"white-space: normal;\">&lt;p&gt;Publication as a Candidate Recommendation does not imply endorsement by the W3C Membership. A Candidate Recommendation Snapshot has received wide review and is intended to gather implementation experience.&lt;/p&gt;</pre><pre xml:space=\"preserve\" style=\"white-space: normal;\">&lt;p&gt;This document was published by the (.)+ Working Group as a Candidate Recommendation Snapshot.&lt;/p&gt;</pre>",
```

Comment:
The text should include how the paragraphs look like and the source code the editors can copy/paste

```suggestion
                            "crIntro": "It <span class=\"rfc2119\">must</span> include the following two paragraphs in \"Status Of This Document\": <blockquote><p>Publication as a Candidate Recommendation does not imply endorsement by the W3C Membership. A Candidate Recommendation Snapshot has received <a href=\"https://www.w3.org/2020/Process-20200915/#dfn-wide-review\">wide review</a> and is intended to gather implementation experience.</p></blockquote><blockquote><p>This document was published by the @@@WG Name@@@ Working Group as a Candidate Recommendation Snapshot.</p></blockquote><div class=\"source\"> <span style=\"font-style: italic\">Include this source code</span>:<br><code>&lt;p&gt;Publication as a Candidate Recommendation does not imply endorsement by the W3C Membership. A Candidate Recommendation Snapshot has received &lt;a href\"https://www.w3.org/2020/Process-20200915/#dfn-wide-review\"&gt;wide review&lt;/a&gt; and is intended to gather implementation experience.&lt;/p&gt;<br>&lt;p&gt;This document was published by the @@@WG Name@@@ Working Group as a Candidate Recommendation Snapshot.&lt;/p&gt;</code></div>",
```